<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewArticle</class>
 <widget class="QWidget" name="ViewArticle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="11" column="0">
      <widget class="QLabel" name="parserLabel">
       <property name="font">
        <font>
         <family>Trebuchet MS</family>
        </font>
       </property>
       <property name="text">
        <string>Parser used: </string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="journalLabel">
       <property name="font">
        <font>
         <family>Trebuchet MS</family>
        </font>
       </property>
       <property name="text">
        <string>Journal:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="articleLabel">
       <property name="font">
        <font>
         <family>Trebuchet MS</family>
         <pointsize>13</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Article:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QTableView" name="tableView">
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item row="5" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="monthLabel">
         <property name="font">
          <font>
           <family>Trebuchet MS</family>
          </font>
         </property>
         <property name="text">
          <string>Month:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="yearLabel">
         <property name="font">
          <font>
           <family>Trebuchet MS</family>
          </font>
         </property>
         <property name="text">
          <string>Year:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="issueLabel">
         <property name="font">
          <font>
           <family>Trebuchet MS</family>
          </font>
         </property>
         <property name="text">
          <string>Issue: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="volumeLabel">
         <property name="font">
          <font>
           <family>Trebuchet MS</family>
          </font>
         </property>
         <property name="text">
          <string>Volume: </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <widget class="Line" name="line_2">
       <property name="font">
        <font>
         <family>Trebuchet MS</family>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="Line" name="line_3">
       <property name="font">
        <font>
         <family>Trebuchet MS</family>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="Line" name="line">
       <property name="font">
        <font>
         <family>Trebuchet MS</family>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Trebuchet MS</family>
        </font>
       </property>
       <property name="text">
        <string>Authors:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
